src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java

```diff
@@ -525,27 +568,32 @@ public void onPotionSplashEvent(final PotionSplashEvent event) {
     }
 
     /**
-     * @param event
+     * HangingPlaceEvent.
+     * @param event HangingPlaceEvent
      */
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onHangingPlaceEvent(final HangingPlaceEvent event) {
         // Cancel the event if a wand is in use
-        if (WandUtils.playerUsesWandOnClick(event.getPlayer(), event.getEntity().getLocation())) {
-            event.setCancelled(true);
-            return;
+        if (event.getPlayer() != null) {
+            if (WandUtils.playerUsesWandOnClick(event.getPlayer(), event.getEntity().getLocation())) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)